org.musescore.MuseScore.yaml

```
@@ -90,6 +90,8 @@ modules:
       - -DMUE_BUILD_CRASHPAD_CLIENT=OFF
       # disable in-app updater
       - -DMUE_BUILD_UPDATE_MODULE=OFF
+      # See: https://github.com/musescore/MuseScore/commit/696279e362afe72db5e92f8a47aa64b3a0e86a86
+      - -D MUE_COMPILE_USE_SYSTEM_HARFBUZZ=ON
```

```suggestion
      - -DMUE_COMPILE_USE_SYSTEM_HARFBUZZ=ON
```

Just for consistency.

---

beside this, LGTM

---

fixed